### PR TITLE
Language class

### LINKED ISSUE #66: Create a Language class
We have lots of language-related configs/logic all over the place and it'd be nice to define this all in a single place using a Language class. Some instance properties of the class should be:

- 2 char language code (eg `ja`)
- emoji flag (eg 🇯🇵)
- native name (eg `日本語` -- just guessing based on Google translate, we'll need the Translation team to help with these)
- lang/region code (eg `ja-JP`)

### PR DESCRIPTION
**Pull request by rviscomi**
Closes #66

### CODE REVIEW

File: src/language.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# coding=utf-8
+class _Language(object):
+  def __init__(self, local_name, lang_code, region_code, emoji_flag):
+    self._local_name = local_name
+    self._lang_code = lang_code
+    self._region_code = region_code
```
Comment (AymenLoukil) [inline src/language.py]: Why do we handle region code ? For countries flags ? 
If it is, language switcher best practices don't recommend displaying countries flags when targeting generic languages.
Example : English speaking people outside US will be confused to see the US flag in the language selector.

Comment (rviscomi) [reply src/language.py]: Because the content is written in American-style English. For example we would spell "color" rather than "colour".

Comment (rviscomi) [reply src/language.py]: In any case, I'm happy to follow up with UX improvements to the language switcher if we're not adhering to best practices. Feel free to file a new issue. cc @tanhengyeow

Comment (AymenLoukil) [reply src/language.py]: The lang in hreflang (or in html lang) is about to say which population am i targeting with this page : 
- Language speaking (en, fr, ja, es...etc) ? 
- Language speaking who are based somewhere (en-us, en-uk, fr-ca).
Search engines use these informations (Google uses hreflang, Bing uses html lang) to provide the page to users based on ths targeting. In our case i believe we should keep on generic language targeting.

Comment (rviscomi) [reply src/language.py]: Gotcha, thanks for explaining. Could you file a new issue so we can discuss those changes? For now, it shouldn't affect this PR, which is mostly just code organization.

File: src/templates/base.html [hunk 2]
```diff
@@ -33,18 +37,12 @@
     </div>
     {% set view_args = request.view_args.copy() %}
```
Comment (AymenLoukil) [inline src/templates/base.html]: the declaration in line 28 is sufficient ?

Comment (rviscomi) [reply src/templates/base.html]: Resolved this by pulling out the view_args logic into a Python function that we can call from the templates.

Comment (AymenLoukil) [reply src/templates/base.html]: Great !

**Pull request merged**